website/source/docs/platform/k8s/upgrading.html.md

```diff
@@ -61,3 +61,81 @@ With the servers upgraded, it is time to upgrade the clients. To upgrade
 the clients, set the `client.image` value to the desired Consul version.
 Then, run `helm upgrade`. This will upgrade the clients in batches, waiting
 until the clients come up healthy before continuing.
+
+## Configuring TLS on an Existing Cluster
```

Comment:
I'd like to see this as a separate page. What do you think? You could link from here to the page.

Reply:
Yeah, I was on the fence about that, but could definitely split.